packages/datadog-plugin-mariadb/src/index.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const Plugin = require('../../dd-trace/src/plugins/plugin');
+const { storage } = require('../../datadog-core');
+const analyticsSampler = require('../../dd-trace/src/analytics_sampler');
+
+class MariadbPlugin extends Plugin {
```

Comment:
The plugin needs a corresponding test. Since MariaDB is very close to MySQL, it should be possible to simply copy over the `mysql` or `mysql2` tests and update them to work with `mariadb`, along with a corresponding entry in `plugins.yml` for GitHub Actions.